src/koans/struct.rs

```diff
@@ -0,0 +1,74 @@
+// Structs are a convenient way to wrap up related data into one neatly packaged structure.
+// Let's consider the example of a Struct that allows us to package together data about a person.
+#[test]
+fn our_first_struct() {
+  struct Person {
+    name: &'static str,
+    age: u32
+  }
+
+  let jim = Person { name: "Jim", age: 57 };
+
+  assert_eq!(jim.name, __);
+  assert_eq!(jim.age, __);
+}
+
+// Let's try another example
+#[test]
+  let movie = Movie { title: "Star Wars", runtime: 121 };
+
+  assert_eq!(movie.title, "Star Wars");
+  assert_eq!(movie.runtime, 121);
+}
+
+// Structs, like anything in Rust, are immutable by default.
+// If we create a mutable instance of a Struct, we can reassign its attributes.
+#[test]
+fn mutable_structs() {
+  struct Language {
+    version: f32
+  }
+
+  let mut rust = Language { version: 1.3 };
+
+  __ = 1.4;
+
+  assert_eq!(rust.version, 1.4);
+}
+
+// We can also decide to temporarily allow a Struct to be mutable
+#[test]
+fn revoking_mutability() {
+  struct Language {
+    version: f32
+  }
+
+  let mut rust = Language { version: 1.3 };
+
+  rust.version = 1.4;
+
+  assert_eq!(rust.version, 1.4);
+
+  let rust = rust;
+
+  rust.version = 1.5;
+
+  assert_eq!(rust.version, 1.5);
+}
+
+// There may be cases where you want to create a new instance of a Struct
+// that is only slightly different from an existing one
+#[test]
+fn dot_merging() {
+  struct Account {
+    holder: &'static str,
+    balance: f64
```

Comment:
Add a third field (like account number), that's long and obnoxious, it'll make the advantages of the `.. foo` syntax more apparent.